source/Hex.lpi

```diff
@@ -68,12 +68,12 @@
           <Version Value="11"/>
           <PathDelim Value="\"/>
           <Target>
-            <Filename Value="../bin/$(TargetCPU)-$(TargetOS)/Hex"/>
+            <Filename Value="..\bin\Hex"/>
```

Comment:
As above...

Reply:
It wasn't me who changed this, it was Lazarus (Fedora). Sorry.

Reply:
When I change it back to the way how I want it to be, will this break the CI/CD actions? If yes, what would I have to change?

Reply:
It's not break the CI/CD!